snuba/datasets/events_processor_base.py

```diff
@@ -29,26 +31,42 @@
 logger = logging.getLogger(__name__)
 
 
+EventData = Mapping[str, Any]
+
+
+class InsertEvent(TypedDict):
```

Comment:
This is named as such just to delineate it from replacement events that come through the same consumer.

Reply:
Also worth noting this is reverse engineered from [`SnubaProtocolEventStream`](https://github.com/getsentry/sentry/blob/cb3a13232991550fd83d584630ede6529046c555/src/sentry/eventstream/snuba.py#L110-L124), so it does not include `search_message` which is referenced below and causes a mypy error.